bin/update-docs.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+require 'active_support/core_ext/hash/deep_merge'
+
+# sed command implementation is different for GNU and macOS
+def sed_i(file)
+  if /darwin/.match?(RUBY_PLATFORM)
+    system "sed -i '' -E 's/[ '$'\t'']+$//' #{file}"
+  else
+    system "sed -i 's/[ \t]*$//' #{file}"
+  end
+end
+
+if ENV['PARALLEL_TESTS_CONCURRENCY']
```

Comment:
with this, by default we run the tests sequentially. do we really want that? I actually don't see a reason why wouldn't we use parallelism all the time

Reply:
also, maybe better to hardcode it to 8 (or at least as the default) so it behaves the same as the CI?